cluster/log-dump/log-dump.sh

```diff
@@ -204,7 +204,7 @@ function save-logs() {
 function export-windows-docker-event-log() {
     local -r node="${1}"
 
-    local -r powershell_cmd="powershell.exe -Command \$log=\$(Get-EventLog -LogName Application -Source Docker); Set-Content '${WINDOWS_LOGS_DIR}\\docker.log' \$log.Message"
+    local -r powershell_cmd="powershell.exe -Command \$logs=\$(Get-EventLog -LogName Application -Source Docker); foreach (\$log in \$logs) {Add-Content '${WINDOWS_LOGS_DIR}\\docker.log' \$(Write-Output \$log.TimeGenerated \$log.Message)}"
```

Comment:
Iterating through the logs and appending seems a little bit slow in my test. I think we can get this faster by filtering directly. For example,
```
$log=(get-eventlog -logname application -source docker | Select-Object -Property TimeGenerated,EntryType,Message);
```

Reply:
Done. The `EntryType` isn't needed here, I just keep the `TimeGenerated` and `Message` in here.

Reply:
Hmm....on second thought, the EntryType might be useful since it shows "Error", "Warning", etc. Do you mind adding that as well?

Reply:
Makes sense! I'll prepare another PR to add that field since this issue if for adding timestamp.

Reply:
I think this PR is small enough that we can get both changes in. Otherwise, we'd have to bother approvers again :-)

Reply:
Sounds good! Added.